### ISSUE TITLE
PaperSpigot: NullPointerException on fast actions

### ISSUE THREAD

**Issue by mibby** [SBPrime/AsyncWorldEdit-Premium#85]
AsyncWorldEdit Premium 3.3.12
WorldEdit dev 3583

Noticed this error going through my logs.

```
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] ***********************************
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] Error performing safe operation for world cx: -357 cz: 146 Loaded: true
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] ***********************************
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * Exception: java.lang.NullPointerException
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * Error message: null
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * Stack: 
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_10_R1.World.getCapturedBlockType(World.java:838)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_10_R1.World.getType(World.java:823)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_10_R1.WorldServer.getTileEntity(WorldServer.java:136)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * net.minecraft.server.v1_10_R1.WorldServer.getTileEntity(WorldServer.java:128)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * com.sk89q.worldedit.bukkit.adapter.impl.Spigot_v1_10_R1.getBlock(Spigot_v1_10_R1.java:213)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * com.sk89q.worldedit.bukkit.BukkitWorld.getBlock(BukkitWorld.java:405)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.world.AsyncWorld$37.execute(AsyncWorld.java:947)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.world.AsyncWorld$37.execute(AsyncWorld.java:944)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.taskdispatcher.TaskDispatcher.performSafe(TaskDispatcher.java:553)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.world.AsyncWorld.getBlock(AsyncWorld.java:944)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * com.sk89q.worldedit.EditSession.getBlock(EditSession.java:416)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.ThreadSafeEditSession.doGetBlock(ThreadSafeEditSession.java:891)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.ThreadSafeEditSession.access$000(ThreadSafeEditSession.java:110)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.ThreadSafeEditSession$2.execute(ThreadSafeEditSession.java:532)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.ThreadSafeEditSession$2.execute(ThreadSafeEditSession.java:529)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.taskdispatcher.TaskDispatcher.performSafe(TaskDispatcher.java:553)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.ThreadSafeEditSession.getBlock(ThreadSafeEditSession.java:529)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.CancelabeEditSession.getBlock(CancelabeEditSession.java:247)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.world.AsyncWorld.doRegen(AsyncWorld.java:665)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.world.AsyncWorld.access$100(AsyncWorld.java:95)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.world.AsyncWorld$23.task(AsyncWorld.java:638)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.WorldAsyncTask.doRun(WorldAsyncTask.java:72)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.primesoft.asyncworldedit.worldedit.BaseTask.run(BaseTask.java:142)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftTask.run(CraftTask.java:58)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * org.bukkit.craftbukkit.v1_10_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:53)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] * java.lang.Thread.run(Thread.java:745)
[20:57:28] [Craft Scheduler Thread - 287/INFO]: [AsyncWorldEdit] ***********************************
```

The only worldedit command ran prior to the error showing up was this ~14 seconds prior.

```
[20:57:12] [Server thread/INFO]: WorldEdit: mibby (in "world"): /regen - Region: (-5792.0, 40.0, 2197.0) - (-5659.0, 80.0, 2460.0)
```

**Comment by SBPrime** [comment]
I checked my version of spigot and I can't see the getCapturedBlockType in  net.minecraft.server.v1_10_R1.World. class. What version of spigot are you using?

**Comment by mibby** [comment]
I use PaperSpigot. Paper is a fork of spigot with additional fixes, performance tweaks, and new APIs / contributions from those who originally and still do contribute upstream to spigot. It maintains upstream parity to spigot, so should be functionally identical. 

https://paper.readthedocs.io
https://aquifermc.org/

**Comment by SBPrime** [comment]
Functionality: yes
Exact code: no

Please send me the exact version that you are using.

**Comment by mibby** [comment]
As of the error occurrence, build 810.
https://ci.destroystokyo.com/job/PaperSpigot/810/

As of now, build 814.
https://ci.destroystokyo.com/job/PaperSpigot/814/

**Comment by SBPrime** [comment]
Fixed in 3.4.9
